debian/control.in

```diff
@@ -0,0 +1,15 @@
+Source: azure-gdpr
+Section: database
+Priority: optional
+Maintainer: Citus Data <packaging@citusdata.com>
+Uploaders: Citus Data <packaging@citusdata.com>
+Build-Depends: debhelper (>= 9), autotools-dev, flex, postgresql-server-dev-all (>= 158~), libedit-dev, libpam0g-dev, libselinux1-dev, libxslt1-dev
```

Comment:
I'm not familiar with the following:
* libpam0g-dev
* libselinux1-dev
* libxslt1-dev

Please make sure that we need all those three with @serprex.

Reply:
@serprex should we remove those or keep them

Reply:
Removed dependencies. @serprex please test after PR merge